Makefile

```diff
@@ -85,6 +107,12 @@ help:
 	@echo "    help:               Print this usage information."
 	@echo "    man:                Generate all man-pages"
 	@echo
+	@echo "    rpm:                Generate the rpm to install osbuild"
+	@echo "    srpm:               Generate the source rpms"
+	@echo "    rpm-devel:          Generate development rpms without checking consistency"
+	@echo "    container:          Generate a development container with osbuild"
```

Comment:
Maybe call it `container-devel` in case we get a non-development container at some point?

Reply:
I'll try to have one naming scheme if possible. Now I call it `_dev` or `.dev` everywhere, but it's called `rpm-devel` in the line above - not sure how to decide here :-/

Reply:
and the documentation was already outdated and the target is named `container.dev`